app/models/supported_ssg.rb

```diff
@@ -26,6 +26,13 @@ def revision
       raw_supported['revision']
     end
 
+    # Collection of supported SSGs that have ZIP available upstream
+    def available_upstream
+      all.reject do |ssg|
+        ssg.upstream_version&.upcase == 'N/A'
+      end
```

Comment:
I really would have just preferred nil here. It makes the code and the SSG support config nicer, too